syntax = "proto3";

service Aggregator
{
  rpc GetStockData (ClientRequest) returns (stream StockAggUpdate);
}

message ClientRequest
{
  string user_id = 1;
}

message StockAggUpdate
{
  string company = 1;
  double price = 2;
  string timestamp = 3;
}
